src/xdocs/google_style.xml

```diff
@@ -66,6 +66,13 @@
         </p>
       </subsection>
       <subsection name="Coverage table" id="Google_Coverage_table">
+        <p>
+          <b>ATTENTION:</b>
+          Links to config and test in following table reference to latest (not released yet) config,
+          it might be slightly different from what we have in latest release.
+          Please always use config that is embedded to jar.
+          Please be sure to use same version of config as your checkstyle version.
```

Comment:
The last line is redundant if the 2nd to last line says to "always" use the one in the embedded jar.

It seems to me the one sentence isn't an always and this should be an either or. Either use the embedded jar or use a custom version copied from one that matches your checkstyle version.

Reply:
fixed.